omnisharptest/omnisharpIntegrationTests/testAssets/slnWithCsproj/.gitignore

```diff
@@ -0,0 +1,2 @@
+**/obj/
```

Comment:
this is moved

I decided it was getting to be more annoying that it was worth to keep re-using the exact same assets for our tools host and O#.  By duplicating them, we can iterate on the new tests without worrying about breaking O#.